doc/source/_templates/autosummary/class_with_autosummary.rst

```diff
@@ -21,24 +21,23 @@
    .. rubric:: {{ _('Methods') }}
 
    .. autosummary::
-      :toctree:
```

Comment:
@jjyao This is needed to prevent member from getting its own file. This is often not what you want when you "opt in" to `class_with_autosummary`.